Note changes to statusbar items behaviour in extension migration guide (#15556)
jupyterlab/jupyterlab: docs/source/extension/extension_migration.rst

- The notebook scroll container is now ``.jp-WindowedPanel-outer`` rather than ``.jp-Notebook``
- Galata notebook helpers `getCell` and `getCellCount` were updated accordingly

> Visibility of StatusBar elements at high magnifications

Reviewer: ```suggestion
Visibility of ``StatusBar`` elements at high magnifications
```

> Visibility of ``StatusBar`` elements at high magnifications
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Reviewer: ```suggestion
Visibility of ``StatusBar`` elements at high magnifications
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

> - Statusbar items are by default now hidden at high magnification/low resolution to prevent overlap for those using the application at high magnifications.

Reviewer: ```suggestion
- Statusbar items are now hidden by default at high magnification/low resolution to prevent overlap for those using the application at high magnifications.
```

> - An additional ``priority`` property has been added to the function ``statusBar.registerStatusItem`` to allow them to remain visible, intended usage is for small statusbar items that either add functionality that would be particularly useful at high zoom or is inaccessible otherwise.

Reviewer: ```suggestion
- An additional ``priority`` property has been added to the options of ``IStatusBar.registerStatusItem`` method to allow the status bar item to remain visible; the intended usage is for small statusbar items that either add functionality that would be particularly useful at high zoom or is inaccessible otherwise.
```
